### ISSUE TITLE
 UserWarning: tinycudann was built for lower compute capability (86) than the system's (89). Performance may be suboptimal. 

### ISSUE THREAD

**Issue by Islaster** [NVlabs/neuralangelo#97]
UserWarning: tinycudann was built for lower compute capability (86) than the system's (89). Performance may be suboptimal.

What does this warning mean?

**Comment by saurabheights** [comment]
It's a warning, not an error. You can ignore it. It's basically checking that you have better gpu card but code is compiled for older GPU cards ( in layman terms). So if you compile for CC 89, the code will be faster.

**Comment by Miaosheng1** [comment]
i also got this warning. it seems my performance have stuck into suboptimal results. 
My CUDA Version is 12.0 Running on 4090.
torch version                    2.0.1+cu117
